Sample Data/ASIM/CarbonBlackAuditLogs_CL_Schema.csv

```diff
@@ -0,0 +1,20 @@
+﻿ColumnName,ColumnOrdinal,DataType,ColumnType
+TenantId,0,"System.String",string
+SourceSystem,1,"System.String",string
+MG,2,"System.String",string
+ManagementGroupName,3,"System.String",string
+TimeGenerated,4,"System.DateTime",datetime
+Computer,5,"System.String",string
+RawData,6,"System.String",string
+"flagged_b",7,"System.SByte",bool
+"verbose_b",8,"System.SByte",bool
+"description_s",9,"System.String",string
+"requestUrl_s",10,"System.String",string
```

Comment:
pls add requestUrl_s to additionalFields

Reply:
Done